Enterprise-CMCS/macpro-mdct-mfp
# [CMDCT-312] - R, E, & T Pages 1-8 Revamp

### Description

Bringing dynamic headers and questions to a government form near you

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/dc951ecc-c76e-439b-acc1-1a7842eb9b23


### Related ticket(s)


CMDCT-312

---
### How to test

Create a work plan
Add a target population to test the other case
Create a SAR
Look through pages 1-8 and see that its all there in fancy number fields with slick section headers (sometimes, depending on page). It should [look a lot like this!](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=21-5886&mode=design&t=m5LjoSAhNObt4yQR-0)

---
### Author checklist
